pkg/machinery/extensions/services/services.go

```diff
@@ -31,6 +31,8 @@ type Spec struct {
 	Depends []Dependency `yaml:"depends"`
 	// Restart configuration.
 	Restart RestartKind `yaml:"restart"`
+	// LogToKmsg enables sending service logs to the kernel ring buffer.
+	LogToKmsg bool `yaml:"logToKmsg"`
```

Comment:
we should document this in the https://www.talos.dev/v1.8/advanced/extension-services/#configuration

probably it should be `LogToConsole`, as `kmsg` is a bit confusing (and doesn't apply e.g. to container mode)